coops-ui/tests/unit/Dashboard.spec.ts

```diff
@@ -70,6 +70,18 @@ describe('Dashboard - UI', () => {
       .getAttribute('disabled')).toBeTruthy()
   })
 
+  it('disables standalone filing buttons & toDo filings when there is a future effective filing pending', () => {
+    wrapper.find(FilingHistoryList).vm.$emit('filings-list', [{ 'status': 'PAID' }])
+    wrapper.find(TodoList).vm.$emit('has-blocker-filing', true)
+
+    expect(wrapper.vm.hasBlockerFiling).toEqual(true)
+    expect(wrapper.vm.coaPending).toEqual(true)
+    expect(wrapper.vm.$el.querySelector('#btn-standalone-addresses')
+      .getAttribute('disabled')).toBeTruthy()
```

Comment:
Try to avoid toBeTruthy() and toBeFalsy() since those aren't type specific and might pass for the wrong reason.
Instead I recommend toBe(true) or toBeDefined() or toBeNull() as needed.

Reply:
Great idea, thank you, will update!